README.md

```diff
@@ -1,3 +1,52 @@
+
+Our DNS registrar is [Namecheap](https://namecheap.com) with name server pointed to Digital Ocean, which allows us to use its APIs to update DNS records with our CI process as needed. For example, our website is published to [Dat](https://dat.foundation) and [IPFS](https://ipfs.io), and our CI publishes the latest IPFS content hash of our website as a TXT record on `ipfs-website.ournetworks.ca`.
+
+### Email
+
+Our Networks is a member of [May First](https://mayfirst.coop), which we rely on for email forwarding and our low-traffic [mailing list](https://lists.mayfirst.org/mailman/listinfo/ournetworks) services, among other infrastrusture things. 
+
+Our email addresses are as follow.
+
+| Email address | Purpose                                                                                                          |
+|:--------------|:-----------------------------------------------------------------------------------------------------------------|
+| billets@      | Coordinate community billets                                                                                     |
+| coc@          | Code of Conduct reporting monitored by organizer on CoC duty                                                     |
+| orga@         | Main address used for general communications and account registrations that has a forward list to all organizers |
+
+### Financial accounts
+
+Our Networks uses [Alterna](https://www.alterna.ca) as the main provider for financial services. Sending transactions from this account require two signatures from organizers who are registered with the financial institution.
+
+We use [Brown Paper Tickets](https://www.brownpapertickets.com) for ticket sales of our conferences and PayPal for small international transactions such as paying honoraria.
+
+### Videos
+
+Conference videos are live streamed by volunteers from [Toronto Mesh](https://tomesh.net) using [IPFS Live Streaming](https://github.com/tomeshnet/ipfs-live-streaming). Post-processed video recordings are published to:
+
+| Host                                                                | Description                                                      |
+|:--------------------------------------------------------------------|:-----------------------------------------------------------------|
+| [Alexandria](https://alexandria.tomesh.net)                         | Media host by Toronto Mesh that is accessible over mesh networks |
+| [Internet Archive](https://archive.org/details/@our_networks)       | Hosted on Internet Archive and with torrent available            |
+| IPFS on [Infura](https://infura.io)                                 | Pinned on infrastructure provided by Infura                      |
+| [YouTube](https://www.youtube.com/channel/UCudGRFTjVGsrKF0h27GER2g) | YouTube playlists                                                |
+
+### Other accounts & tools
```

Comment:
Could we use the service inventory @patcon has helped refine elsewhere? 

I think it could be a separate page, also this is also non-exhaustive.

Reply:
@dcwalk @patcon can you link the service inventory?

I think these should all be in separate page and linked to from README, but I rather do that as a post-conference discussion. By then we'll have better clarity of what needs to be in this repo.

Reply:
- https://link.hypha.coop/inventory
- https://link.civictech.ca/inventory
- (edgi service inventory is not public)

Reply:
I am confused about the mixing of ournetworks with hypha, @dcwalk are you referring to what @patcon posted or something else?

Reply:
I believe she's suggesting the pattern of that style of doc, not using the same doc.

Reply:
Yes, just the pattern